integration_test/pg/binary_default_test.exs

```diff
@@ -0,0 +1,28 @@
+defmodule Ecto.Integration.BinaryDefaultTest do
```

Comment:
Let's please not add a whole new module only for this test. Instead, let's add it to `integration_test/sql/migration.exs` and make sure it also passes for MySQL.

Reply:
Any advice on how to make it pass for MySQL? It doesn’t support defaults for binaries.

Reply:
Oh, if it doesn't support default for binaries let's use a tag. Something like this:

    @tag :binary_default_in_migration

And then you can skip this tag in the MySQL test suite. :)